kernel/sched.c

```diff
@@ -1236,8 +1235,10 @@ static int32_t z_tick_sleep(k_ticks_t ticks)
 
 	k_timeout_t timeout = Z_TIMEOUT_TICKS(ticks);
 	if (Z_TICK_ABS(ticks) <= 0) {
+		/*? What is the intention here? uint32_t = int64_t + uint32_t */
```

Comment:
this is going to be lost and forgotten. Please open an issue about these comments.

Reply:
Isn't better first that code owner confirms that this is unexpected?

Reply:
The code owner could not spot that in 1 single +1,273 −1,217 commit.